examples/solids/index.rst

```diff
@@ -704,14 +708,18 @@ using different algebraic manipulations.
    We define a second order *Green-Euler* strain tensor (cf. Green-Lagrange strain :math:numref:`eq-green-lagrange-strain`) as
 
    .. math::
+      :label: green-euler-strain
+
       \bm e = \frac 1 2 \Big(\bm{b} - \bm{I}_3 \Big) = \frac 1 2 \Big( \nabla_X \bm{u} + (\nabla_X \bm{u})^T + \nabla_X \bm{u} \, (\nabla_X \bm{u})^T \Big).
 
    Then, the Kirchhoff stress tensor :math:numref:`tau-neo-hookean` can be written as
 
    .. math::
+      :label: tau-neo-hookean-stable
+      
       \bm \tau = 2\mu \bm e + \lambda \log J \bm I_{3},
```

Comment:
Let's just be consistent in ordering between this and `neo-hookean-stress-stable`. I'm not opposed to the `2 \mu \bm e` (cf. `2 \mu \bm C^{-1} \bm E`) term coming first, but we should be consistent and we've used the other convention up above. So either edit the context so it flows consistently in both cases or switch this.

Reply:
```suggestion
      \bm \tau = \lambda \log J \bm I_{3} + 2\mu \bm e,
```

Reply:
Yeah, you are right. Thanks for the commit.

Reply:
I think we can merge it after the test passed, right?